packages/components/src/tabs/test/index.tsx

```diff
@@ -1835,25 +1321,11 @@ describe( 'Tabs', () => {
 
 				// re-enable all tabs
 				rerender(
-					<ControlledTabs initialTabId="gamma" selectedTabId="gamma">
-						<Tabs.TabList>
-							{ TABS.map( ( tab ) => (
-								<Tabs.Tab
-									key={ tab.id }
-									id={ tab.id }
-									title={ tab.title }
-									className={ tab.tab.className }
-								>
-									{ tab.title }
-								</Tabs.Tab>
-							) ) }
-						</Tabs.TabList>
-						{ TABS.map( ( tab ) => (
-							<Tabs.TabPanel key={ tab.id } id={ tab.id }>
-								{ tab.content }
-							</Tabs.TabPanel>
-						) ) }
-					</ControlledTabs>
+					<ControlledTabs
+						tabs={ TABS }
+						initialTabId="gamma"
```

Comment:
And here too. Let's make sure all `ControlledTabs` tests don't use `initialTabId` (unless it is a specific test demonstrating that it gets ignored in controlled mode, in favor of `selectedTabId`)